solutions/src/main/java/ru/sberbank/school/task02/ConversionService.java

```diff
@@ -0,0 +1,90 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+
+public class ConversionService implements FxConversionService {
+
+    private ExternalQuotesService externalQuotesService;
+
+    public ConversionService(ExternalQuotesService externalQuotesService) {
+        this.externalQuotesService = externalQuotesService;
+    }
+
+    @Override
+    public BigDecimal convert(ClientOperation operation, Symbol symbol, BigDecimal amount) {
+        try {
+            if (operation == null || symbol == null || amount.compareTo(BigDecimal.ZERO) < 0 || amount == null) {
+                throw new IllegalArgumentException();
+            }
+        }
```

Comment:
&#39;}&#39; at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a812194582>*